README.md

```diff
@@ -186,6 +191,18 @@ Path to Android NDK []: C:\tools\android-ndk
 Path to Java JDK [C:\Program Files\Java\jdk1.8.0_131]:
 ```
 
+#### Setup AppleTV
+- Run `lime rebuild hxcpp tvos`
+- Remove stable lime `haxelib remove lime`
+- Install lime from GitHub `git clone --recursive https://github.com/openfl/lime`
```

Comment:
What is the reason for this? Do they have a fix in git verson after 5.2.1 that fixes tvos? I didnt see anything special related to that in their changelog